### PR TITLE
[av] Add GCS destination to data integration pipeline.

### PR DESCRIPTION
**Pull request by andreventura02**
# Summary

Add GCS destination to data integration pipeline.

Supported file types

- parquet
- csv

# Tests
tested locally

cc: @wangxiaoyou1993

### CODE REVIEW

**Comment by wangxiaoyou1993** [comment]
Thanks! Will review the PR today.

File: mage_integrations/mage_integrations/destinations/google_cloud_storage/README.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Google Cloud Storage
+
+![]()
```
Comment (wangxiaoyou1993) [inline mage_integrations/mage_integrations/destinations/google_cloud_storage/README.md:3]: Could you find an image for the Google Cloud Storage and add here?

File: mage_integrations/mage_integrations/destinations/google_cloud_storage/templates/config.json [hunk 2]
```diff
@@ -0,0 +1,8 @@
+{
+  "bucket": null,
+  "file_type": "parquet",
+  "object_key_path": null,
+  "table": null,
+  "date_partition_format": null,
+  "google_application_credentials": "/path/to/your/service/account/key.json"
+}
```
Comment (wangxiaoyou1993) [inline mage_integrations/mage_integrations/destinations/google_cloud_storage/templates/config.json:8]: nit: add an empty line

File: mage_integrations/mage_integrations/destinations/google_cloud_storage/__init__.py [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+        client = self.build_client()
+        client.get_bucket(self.bucket)
+
+
+if __name__ == '__main__':
+    destination = GoogleCloudStorage(
+        argument_parser=argparse.ArgumentParser(),
+        batch_processing=True,
+    )
+    destination.process(sys.stdin.buffer)
```
Comment (wangxiaoyou1993) [inline mage_integrations/mage_integrations/destinations/google_cloud_storage/__init__.py:93]: nit: add an empty line

File: mage_integrations/requirements.txt [hunk 4]
```diff
@@ -8,6 +8,7 @@ google-analytics-data==0.14.2
 google-api-python-client==2.70.0
 google-cloud-bigquery-storage==2.16.2
 google-cloud-bigquery~=3.0
+google-cloud-storage==2.8.0
```
Comment (wangxiaoyou1993) [inline mage_integrations/requirements.txt:11]: Let's use version 2.5.0 here to make it consistent with this file: https://github.com/mage-ai/mage-ai/blob/master/requirements.txt#L51

File: mage_integrations/mage_integrations/destinations/google_cloud_storage/__init__.py [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+    def file_type(self) -> str:
+        return self.config.get('file_type')
+    
+    @property
+    def object_key_path(self) -> str:
+        return self.config['object_key_path']
+    
+    def build_client(self) -> Client:
+        return Client.from_service_account_json(
+            json_credentials_path=self.config['google_service_acc_key_filepath']
```
Comment (wangxiaoyou1993) [inline mage_integrations/mage_integrations/destinations/google_cloud_storage/__init__.py:28]: Let's make sure the this key name is same with the one in the template config.json

File: mage_integrations/mage_integrations/destinations/google_cloud_storage/README.md (hunk 1 above)
Comment (andreventura02) [reply mage_integrations/mage_integrations/destinations/google_cloud_storage/README.md:3]: I looked for an image, but I didn't find it

**Review by wangxiaoyou1993 — APPROVED** [review]

**Pull request merged**